return function()
	describe("When an error occurs in an it block", function()
		it("Should error", function()
			error("Failure in it block")
		end)
		afterEach(function()
			error("afterEach threw an error as expected")
		end)
	end)
end
